design-components/packages/railz-components-react/package.json

```diff
@@ -46,7 +46,7 @@
     "minimist": "1.2.6"
   },
   "peerDependencies": {
-    "@railzai/railz-uikit": "^0.0.0",
+    "@railzai/railz-uikit": "^0.0.1-alpha.1",
```

Comment:
This could be:
"@railzai/railz-uikit": "latest"

That way we don't need to update this every release. This was auto-generated?